dmutils/api_stubs.py

```diff
@@ -15,7 +15,8 @@ def _derive_framework_family(slug):
 def lot(lot_id=1, slug="some-lot", name=None, allows_brief=False, one_service_limit=False, unit_singular='service',
         unit_plural='services'):
     if not name:
-        name = slug.replace("-", " ").title()
+        name = slug.replace("-", " ")
+        name = name[:1].upper() + name[1:]
```

Comment:
How about `name = slug.capitalize().replace("-", " ")`

Reply:
That's exactly what I wanted. 🤦‍♂️